ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py

```diff
@@ -246,7 +301,7 @@ class os_service(LinuxCommandPlugin):
                 'sudo initctl list 2>&1 || true ; '
                'elif command -v service >/dev/null 2>&1; then '
                 'echo "SYSTEMV"; '
-                'sudo service --status-all 2>&1 || true; '
+                'sudo ls -l /etc/rc$(/sbin/runlevel | sed -r "s/[^0-9]//g").d/ 2>&1 || true; '
```

Comment:
The sed expression used here seems wrong according to runlevel's help.

> $ runlevel --help
> runlevel [OPTIONS...]
> 
> Prints the previous and current runlevel of the init system.

My current example..

```
$ runlevel
N 5
```

So that sed expression will work in this (probably most common) case, but the N could be a number according to the help, and the expression would fail in that case. You may want to use `awk {print $2}` instead of sed to make sure you're only capturing the current runlevel.